packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -266,7 +269,7 @@ export default class QuanticRefineModalContent extends LightningElement {
    * @returns {Array<SortOption>}
    */
   get sortOptions() {
-    return this.hasSortOptions ? this.sortData : [];
+    return this.isSortComponentReady ? this.sortData : [];
```

Comment:
we don't need `this.isSortComponentReady` in this condition.
cause the getter `sortOptions` is read only when `hasSortOptions` is true.

Reply:
this was not part of the last commit, please recheck.

also this could be adjusted to the following:
```
  get sortOptions() {
    return this.sortData?.length ? this.sortData : []  
  }
```

This way you would no longer need the `hasSortOptions` getter.

Reply:
yeah :D addressed that